Header header
int32 id
int32 uav
int32 grafo
int32[] edges
float32 tx
float32 ty
float32 theta
float32 scale
float32 theta_offset
float64 latitude
float64 longitude
float64 altitude
float64 heading
float64 pitch
float64 roll
string camera_type
string  filename
sensor_msgs/CompressedImage image
